packages/core/src/lib/jdk/JdkHelper.ts

```diff
@@ -50,6 +51,18 @@ export class JdkHelper {
     }
   }
 
+  /**
+   * Runs the `java` command, passing args as parameters.
+   */
+  async runJava(args: string[]): Promise<{stdout: string; stderr: string}> {
+    const java = this.process.platform === 'win32' ? '/bin/java.exe' : '/bin/java';
```

Comment:
Tangential question - does this cover 64 bit versions of Windows too?

Reply:
It does: https://nodejs.org/api/os.html#os_os_platform

`win32` is a bad name. To check for 32bit / 64bit one would use `os.arch()`(https://nodejs.org/api/os.html#os_os_arch)

Reply:
Ah cool - this wasn't blocking, I was just curious.

Feel free to merge!